### ISSUE TITLE
GATK4_CREATESEQUENCEDICTIONARY memory misspecification

### ISSUE THREAD

**Issue by mjakobs** [nf-core/sarek#1085]
Hi team, thanks for releasing sarek version 3.2.1!  

While running the test profile (`nextflow run nf-core/sarek -r 3.2.1 -profile test,singularity --outdir results -c my.config`) I found that `GATK4_CREATESEQUENCEDICTIONARY` fails with `Error: A JNI error has occurred, please check your installation and try again`.  It appears the module accidentally specifies 6MB memory rather than 6GB for this task.  
Relevant code:
https://github.com/nf-core/sarek/blob/6c0d335e17fb4406f527540631da7b26a5fe1464/modules/nf-core/gatk4/createsequencedictionary/main.nf#LL22C1-L30C72

Proposed downstream fix would be to change the module.  In the meantime, adding the following to my config file has worked for me:
```
withName: 'GATK4_CREATESEQUENCEDICTIONARY' {
        memory       = { 6.GB * task.attempt }
    }
```

**Comment by asp8200** [comment]
I suspect that there is a similar problem in `GATK4_COMPOSESTRTABLEFILE`.

**Comment by maxulysse** [comment]
It's due to https://github.com/nf-core/modules/pull/3027,
There are actually more modules to fix, thanks a lot @mjakobs for spotting that

**Comment by maxulysse** [comment]
Will be fixed on the modules side by https://github.com/nf-core/modules/pull/3502

**Comment by maxulysse** [comment]
and will be closed by #1087
